webpack.config.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
```

Comment:
- [ Required ] I think it would be better if you don't disable linter checks in your project, linter checks help developers to write clean and professional code disabling them is not considered good practice. Please consider fixing this linter error.

Reply:
Thanks for your feedback dear **CR,**

I thought that Linter checks may be skipped into configurations files to not broke this, for instance in a previous [lesson](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md). do you @Felix45  ?

Thanks